src/task2/service/userService.js

```diff
@@ -0,0 +1,70 @@
+const userRepository = require('./../repository/usersRepository');
+const sharedConstants = require('./../util/sharedConstants');
+const { User } = require('./../data/User');
+
+async function createUser(payload) {
+    let user = await userRepository.getUser(payload.id);
+    if (!user) {
+        user = new User(payload);
+        return userRepository.saveUser(user);
+    }
+    throw new Error(sharedConstants.USER_ALREADY_EXISTS_MESSAGE);
+}
+
+async function updateUser(id, userData) {
+    const user = await userRepository.getUser(id);
+    if (user) {
+        updateUserWithNewData(user, userData);
+        return await userRepository.saveUser(user);
```

Comment:
`return await` is redundant, please be consistent and use just `return` instead

Reply:
Fixed